src/schemathesis/models.py

```diff
@@ -178,8 +179,12 @@ def call(
             close_session = False
 
         data = self.as_requests_kwargs(base_url)
+        if "headers" not in data or data["headers"] is None:
```

Comment:
`"headers" not in data` is redundant, since the `as_requests_kwargs` method always returns a dictionary with "headers" included